2024-11-26 18:00:04,757 INFO    MainThread:180958 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-11-26 18:00:04,757 INFO    MainThread:180958 [wandb_setup.py:_flush():79] Configure stats pid to 180958
2024-11-26 18:00:04,757 INFO    MainThread:180958 [wandb_setup.py:_flush():79] Loading settings from /home/onebom/.config/wandb/settings
2024-11-26 18:00:04,757 INFO    MainThread:180958 [wandb_setup.py:_flush():79] Loading settings from /data/onebom/project/point_based_vp/ongoing/conditional_vp/wandb/settings
2024-11-26 18:00:04,757 INFO    MainThread:180958 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-11-26 18:00:04,757 INFO    MainThread:180958 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-11-26 18:00:04,757 INFO    MainThread:180958 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program_abspath': '/data/onebom/project/point_based_vp/ongoing/conditional_vp/train.py', 'program': '/data/onebom/project/point_based_vp/ongoing/conditional_vp/train.py'}
2024-11-26 18:00:04,757 INFO    MainThread:180958 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-11-26 18:00:04,757 INFO    MainThread:180958 [wandb_init.py:_log_setup():534] Logging user logs to /data/onebom/project/point_based_vp/ongoing/conditional_vp/wandb/run-20241126_180004-qjt0ttra/logs/debug.log
2024-11-26 18:00:04,758 INFO    MainThread:180958 [wandb_init.py:_log_setup():535] Logging internal logs to /data/onebom/project/point_based_vp/ongoing/conditional_vp/wandb/run-20241126_180004-qjt0ttra/logs/debug-internal.log
2024-11-26 18:00:04,758 INFO    MainThread:180958 [wandb_init.py:init():621] calling init triggers
2024-11-26 18:00:04,758 INFO    MainThread:180958 [wandb_init.py:init():628] wandb.init called with sweep_config: {}
config: {}
2024-11-26 18:00:04,758 INFO    MainThread:180958 [wandb_init.py:init():666] wandb.init() called when a run is still active
2024-11-26 22:02:25,578 WARNING MsgRouterThr:180958 [router.py:message_loop():77] message_loop has been closed
